src/software/ai/navigator/navigator_test.cpp

```diff
@@ -3,18 +3,19 @@
 #include <gtest/gtest.h>
 
 #include "software/ai/intent/all_intents.h"
+#include "software/ai/navigator/path_planner/theta_star_path_planner.h"
 #include "software/ai/primitive/all_primitives.h"
 #include "software/test_util/test_util.h"
 
 TEST(NavigatorTest, convert_catch_intent_to_catch_primitive)
 {
     World world = ::Test::TestUtil::createBlankTestingWorld();
-    Navigator pathplanningNavigator;
+    Navigator Navigator(std::make_unique<ThetaStarPathPlanner>(ThetaStarPathPlanner()));
```

Comment:
`Navigator` -> `navigator` for variable naming, same goes for the rest of this file. I'm kinda shocked that the compiler lets us declare a variable with the same name as a type...........

Reply:
:o

Reply:
Done